### ISSUE TITLE
Text Contrast Errors

### ISSUE THREAD

**Issue by smarziano** [smartchicago/chicago-atlas#294]
There are many places on the Chicago Health Atlas beta, where the text is very low contrast and doesn't meet accessibility standards. Listed below is a checklist of the colors used on a white background (unless otherwise noted) that should be changed to make it easier for users to read.

*Homepage*
- [ ] Search box text: #909f9f
- [ ] "Learn more" text: #fd4141
- [ ] Footer text: #6b7d8f

*Healthy Chicago*
- [ ] All chart text, ex: "Overall Health Status": #727e82
- [ ] Linked data source text: #fd4141 

*Topics page*
- [ ] Linked topics text: #fd4141 

*Specific indicator page* _(example: /youth-smoking)_
- [ ] Chosen tab "Summary"  #fd4141 
- [ ] Non-selected tab (ex: "Disparities") #727e82
- [ ] Text under "Understanding the Measure" #6b7d8f
- [ ] Left-hand nav bar: Foreground color #7e95ac on background #f6f7fa
- [ ] Left-hand nav bar title "Topics:" Foreground: #6b7d8f on background #f6f7fa

*Community areas page*
- [ ] Community area names: #fd4141

*Specific community page* _(example: /roseland)_
- [ ] Zip code buttons: #ffffff on #fd4141
- [ ] Gender % text: #727e82 on #ffffff
- [ ] Race/Ethnicity bar graph: 1) #ffffff on #5286b9, 2) #ffffff on #769fc8, 3) #ffffff on #9ab9d7, 4) #ffffff on #bfd2e5

Note: For some reason, my contrast checker tool had a difficult time reviewing the /resources page. Therefore, I would just check for similar color uses and themes that were identified on other pages to guide the redesign of this page, too.

**Comment by jcrichlow** [comment]
Hi @smarziano can you add the "enhancement" label to this issue?

**Comment by jcrichlow** [comment]
@parrolabs will migrate LOEs from Asana

**Comment by jcrichlow** [comment]
LOE: Discovery session needed
